src/Commands/MakeDataTransferObjectFactoryCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Commands;
+
+use Regnerisch\LaravelBeyond\Resolvers\AppNameSchemaResolver;
+use Regnerisch\LaravelBeyond\Resolvers\DomainNameSchemaResolver;
+
+class MakeDataTransferObjectFactoryCommand extends BaseCommand
```

Comment:
Maybe we should think about creating a DTO and a DTO factory by one command? Or at least provide a flag here for creating a DTO.

Reply:
Feel free to add the required code :)

Reply:
I'll provide a separate PR for this feature.